packages/core/src/ObjectContainer.js

```diff
@@ -504,9 +515,50 @@ export default class ObjectContainer {
       }
     }
 
+    if (this._isSpread(name) && entry?.sharedInstance.length) {
+      let spreadEntry = Entry.createFromEntry(entry);
+
+      spreadEntry.sharedInstance = entry.sharedInstance.map(sharedInstance =>
+        this.get(sharedInstance)
+      );
+
+      this._entries.set(name, spreadEntry);
```

Comment:
Why we set `spreadEntry` in entries? If I call `getEntry` two times with 
'...Spread Constant' I don't use spreadEntry from previous call.

Reply:
you're correct, it slips into this if at all times so there should be some kind of check for this... or we should probably no even put it into the entries map.